css-brochure-form - Paolo Di Martino

Paolo-DM opened the pull request:
![SCR-20220316-wln](https://user-images.githubusercontent.com/69313887/158702148-5dcd712a-c3f6-40c4-b97e-70e64b4913e6.png)

TimelikeClosure reviewed (changes requested):
Almost there, the main difference is that the page heading should be smaller and have a higher font weight:

---

![image](https://user-images.githubusercontent.com/16143771/158702451-a469dab0-79c0-425c-8dfe-dfa037926b95.png)

---

![image](https://user-images.githubusercontent.com/16143771/158702495-09b79967-6b90-4316-967b-2de3a7f4d385.png)

Paolo-DM commented:
Changed the heading from h2 to h3 and added the font-weight: bold property.

![SCR-20220316-wz7](https://user-images.githubusercontent.com/69313887/158703602-14575e9a-0ee3-4416-badb-b3a11edd46bc.png)
